src/artworks/artworks.repository.ts

```diff
@@ -27,6 +34,16 @@ export class ArtworksRepository {
         this.artworks = this.firebaseRepository.firestore.collection('artworks');
     }
 
+    async createArtwork(artwork: NewArtwork) {
+        const newArtwork: _.Omit<FirebaseArtwork, 'id'> = {
+            title: artwork.title,
+            imageUrl: artwork.imageUrl,
+            eventbriteId: artwork.eventbriteId,
+            artist: this.peopleRepository.people.doc(artwork.artistId)
+        };
+        await this.artworks.add(newArtwork);
```

Comment:
For the image upload, it will be good to deal with file type instead of image url and upload from api rather than through frontend.